### PR TITLE
Motion config: 'calculate % without abstains' (Fixes #2182) 

### LINKED ISSUE #2182: Add new config option for 100 % base of voting results (yes+no without abstain)
Add to the config field "The 100 % base of a voting result consists of" (in section motions and elections) a new select option:
"Only YES and NO values"

Add a check for this new config option in [motions/base.js](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/motions/static/js/motions/base.js#L86) and [assignments/base.js#L38](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/assignments/static/js/assignments/base.js#L38) and [assignments/base.js#L107](https://github.com/OpenSlides/OpenSlides/blob/master/openslides/assignments/static/js/assignments/base.js#L107).

### PR DESCRIPTION
**Pull request by emanuelschuetze**
Rework on #2192 and added new commit to clean up / style something.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Ready for last review and merge.

File: openslides/motions/static/js/motions/site.js [hunk 1]
```diff
@@ -70,14 +70,15 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
...
                             var id = index + 1,
-                                yes = poll.yes,
-                                yesRelative = (poll.yes) * 100 / (poll.votescast),
-                                no = poll.no,
-                                noRelative = (poll.no) * 100 / (poll.votescast),
-                                abstain = poll.abstain,
-                                abstainRelative = (poll.abstain) * 100 / (poll.votescast),
-                                valid = poll.votesvalid,
-                                validRelative = (poll.votesvalid) * 100 / (poll.votescast),
+                                yes = poll.yes ? poll.yes : 0, // if no poll.yes is given, set it to 0
```
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: Is it possible to use `'-'` instead of `0`. See your solutution for votesvalid.

**Comment by normanjaeckel** [comment]
One question. Then rebase and merge.

File: openslides/motions/static/js/motions/site.js (hunk 1 above)
Comment (emanuelschuetze) [inline openslides/motions/static/js/motions/site.js]: I changed it for poll.yes and poll.no to '-'.

**Comment by emanuelschuetze** [comment]
All done. Ready for merge now.

File: openslides/motions/static/js/motions/site.js (hunk 1 above)
Comment (normanjaeckel) [inline openslides/motions/static/js/motions/site.js]: and abstain ... :-)

**Comment by emanuelschuetze** [comment]
Done also for abstain.

**Pull request merged**